pkg/cluster/cluster.go

```diff
@@ -390,6 +399,161 @@ func (c *Cluster) createPod(members etcdutil.MemberSet, m *etcdutil.Member, stat
 	return err
 }
 
+// createNetPolicy creates a NetworkPolicy for a Sensu cluster
+func (c *Cluster) createNetPolicy() error {
+	label := map[string]string{
+		"sensu_cluster": c.cluster.Name,
+	}
+
+	netCases := []networkingv1.NetworkPolicy{
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				GenerateName: "sensu-block-all-",
+				Labels:       label,
+				Namespace:    metav1.NamespaceDefault,
+			},
+			Spec: networkingv1.NetworkPolicySpec{
+				PodSelector: metav1.LabelSelector{
+					MatchLabels: map[string]string{
+						"app":           "sensu",
+						"sensu_cluster": c.cluster.Name,
```

Comment:
The `MatchLabels` are repeated a few times. Can you define a variable for them, too?